commons/src/main/java/org/infinispan/commons/api/functional/Param.java

```diff
@@ -44,6 +44,29 @@
    P get();
 
    /**
+    * Should this paramater go over the wire?
+    */
+   SerializeMethod serializeMethod();
```

Comment:
I'm afraid I'm responsible for the mixup :) I'll explain what happened so other can learn from my mistakes ;) Basically I should've created separate PR for the backport of this PR to 8.2.x, and then base my "proposed fix" on that. But at first I thought they could be independent of each other so I only created one PR. Later on this proved to be untrue. I did create separate branches for both but it was too late to change the base branch of the PR and the backport shows as part of my PR.